### PR TITLE
Feature/#20 like song button

### LINKED ISSUE #20: Like Song Button
Hi!

I've been looking for a mini Spotify player for Windows since a while and this one seems to fit the bill almost perfectly! I'll start using it!

What I would like to have is a button to like or unlike the currently playing song. Could I start working on that feature?

Thank you and keep up the good work!

### PR DESCRIPTION
**Pull request by GabrielCoteM**
Hello!

This adds a like/unlike button to the main interface, so the user can save and unsave the currently playing track into their Liked Songs library.

It is shown as a new heart icon that is placed at the right of the album art. It's shown as filled if the song is liked, or as an outline if it's not liked, just like in Spotify itself. Clicking on the heart switches the icon and does the saving/unsaving operation on the user's account.

Two scopes had to be added for the feature to work: 
- user-library-read
- user-library-modify

Hope you like it! Thank you!

Fixes #20

### CODE REVIEW

**Review by KieranBond — CHANGES_REQUESTED** [review]
This is looking great! Works like a charm too.

I think this is exactly how I would of implemented it and keeps completely in line with how the project is so I'll approve this after one thing - If you could just remove the changes to the `README.md` and add them perhaps into the Pull Request that would be great.

On the commit where you remove the ReadMe updates, it would be good to mention the scopes being modified too - Just to help keep the Git history useful.

Once that's done, I'm happy to merge this in :)

**Comment by GabrielCoteM** [comment]
Alright so I brought the README.md file like before.

Thanks for the feedback! Much appreciated!

**Review by KieranBond — APPROVED** [review]

**Comment by KieranBond** [comment]
Amazing, thanks @GabCM. Feel free to contribute more if there's anything you want added!

**Pull request merged**
